docs/framework/Release-Notes/01-January-Preview-of-Quality-Rollup

```diff
@@ -0,0 +1,57 @@
+<h2>January Preview of Quality Rollup</h2>
+<italics> released January 24, 2024</italics>
```

Comment:
The title is different from previous blog post's title ".NET Framework January 2024 Cumulative Update Preview" (https://devblogs.microsoft.com/dotnet/dotnet-framework-january-2024-cumulative-update-preview/). Is that intentional? Why was the change?